docs/webapp/webapp_model_endpoints.md

```diff
@@ -0,0 +1,90 @@
+* Uptime - Longest duration that any of the model instances has been running
+* \# Requests - Total number of requests to the endpoint
+* Requests / MIN (avg) - Average request rate in the last minute
+* Latency (avg) - Average endpoint response latency
+
+![Active endpoints](../img/webapp_model_endpoints_active_table.png)
+
+The **Loading** tab shows endpoints that are being set up, but are not yet active. The table provides the following 
+information:
+
+* Instance ID - The model deployment application instance ID
+* Model - Model Name
+* Uptime - Time since this endpoint has started setting up
+* Preprocess artifact - Preprocessing code used for the endpoint 
+* Input type - Model matrix input type  (e.g. uint8, float32, int16, float16)
+* Input size - Model matrix input size `[Rows x Columns X Channels etc ...]`
+
+![Loading endpoints](../img/webapp_model_endpoints_loading_table.png)
+
+You can apply column filters by clicking <img src="/docs/latest/icons/ico-filter-off.svg" alt="Filter" className="icon size-md" />, 
+and sort endpoints by clicking <img src="/docs/latest/icons/ico-sort-off.svg" alt="Sort order" className="icon size-md space-sm" /> on the relevant column.
+
+## Active Endpoint Details 
+Clicking on a model endpoint opens it in detailed view.
+
+### Details
+
+The **Details** tab displays the model endpoint information: 
+* Endpoint Name
+* Endpoint URL
+* Model name (click to go to ClearML or HuggingFace model page)
+* Uptime - Duration of longest running endpoint instance
+* Preprocess artifact - Preprocessing code used for the endpoint 
+* Input type - Model matrix input type  (e.g. uint8, float32, int16, float16)
+* Input size - Model matrix input size `[Rows x Columns X Channels etc ...]`
+* Model instances list - Model instances servicing the endpoint. The following information is displayed for each instance:
+  * Instance ID - Click instance ID to go to the model endpoint's ClearML Application instance dashboard .
+  * Uptime
+  * Number of requests - Total count
+  * Request rate - Average requests per minute
+  * CPU Count
+  * GPU Count 
+  * Latency - Average request latency in the last minute
+
+![Endpoints details](../img/webapp_model_endpoints_details.png)
+
+### Monitor
+The Monitor tab displays the endpoint's operational metrics and resource usage over time graphs: 
+* Total number of requests 
+* Average requests/min 
+* Average latency 
+* Machine utilization metrics:
+  * CPU and GPU Usage
+  * Memory Usage 
+  * Video Memory Usage
+  * Network Usage
+
+![Endpoints monitor](../img/webapp_model_endpoints_monitor.png)
+
+The graphs' time span can be controlled through the menu at its top right corner. 
+Click <img src="/docs/latest/icons/ico-show.svg" alt="Eye Show" className="icon size-md space-sm" /> to control which 
+plots to display. Click **HIDE ALL**, and then click <img src="/docs/latest/icons/ico-show.svg" alt="Eye Show" className="icon size-md space-sm" /> on the plots you want to see.
```

Comment:
@pollfly "Click **HIDE ALL**..." should not be an instruction, but a use case example of choosing what to display